src/main/java/com/redhat/red/build/finder/BuildFinder.java

```diff
@@ -403,27 +404,26 @@ public BuildFinder(ClientSession session, BuildConfig config) {
 
         session.close();
 
-        final long endTime = System.nanoTime();
-        final long duration = endTime - startTime;
-        final double seconds = (duration / 1000000000.00);
-        int lookups = builds.size() - 1;
+        final Instant endTime = Instant.now();
+        final Duration duration = Duration.between(startTime, endTime).abs();
```

Comment:
Question - apart from using the new API was there any reason to switch from the system.nanotime call?

Reply:
While I still don't quite like the way the new API looks, the main reason is that the API handles formatting minutes and seconds (in standard format) whereas the old code does not, and I thought that this change makes the log message easier to read when processing particularly large distributions that take a lot of time.

Reply:
Ok, just curious.